libPhoneNumberTests/NBPhoneNumberUtil+ShortNumberTest.h

```diff
@@ -0,0 +1,29 @@
+//
+//  NBPhoneNumberUtil+ShortNumberTest.h
```

Comment:
Rename, possibly...  Seeing "Test" at the end of a class name usually implies it's an actual test case (ie. extends XCTest)

Reply:
Renamed it to NBPhoneNumberUtil+ShortNumberTestHelper